src/reyna.Win32/NativeMethods/NativeMethods.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Reyna
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using Reyna.Power;
+
+    public class NativeMethods
+    {
+        internal static byte ACLineStatus { get; set; }
+
+        public static int GetSystemPowerStatusEx(SystemPowerStatus systemPowerStatus, int update)
```

Comment:
This is for Win32 testing purposes, it allows us to set a mock value of whether the "device" is plugged in or not for use in xunit tests.
@CharlesGarth